packages/core/src/lib/TwaManifest.ts

```diff
@@ -22,16 +22,11 @@ import {findSuitableIcon, generatePackageId, validateNotEmpty} from './util';
 import Color = require('color');
 import Log from './Log';
 import {WebManifestIcon, WebManifestJson} from './types/WebManifest';
+import {ShortcutInfo, SHORT_NAME_MAX_SIZE} from './ShortcutInfo';
```

Comment:
`SHORT_NAME_MAX_SIZE` is also use used for the launcher name, which is unrelated to `ShortcutInfo`, so it's a bit strange to import it here.

I don't think there's an established relationship between the maximum size of the Shortcut short name and the app short name, so we might want to use different variables. Alternatively, we'd define the variable somewhere else and import in both places.